broker/handler.go

```diff
@@ -60,13 +80,30 @@ func (handler *QueryHandler) HandleSQL(w http.ResponseWriter, r *http.Request) {
 		apiCom.RespondWithError(w, err)
 		return
 	}
-	// TODO: logging and metrics
 	return
 }
 
 func (handler *QueryHandler) HandleAQL(w http.ResponseWriter, r *http.Request) {
 	var queryReqeust BrokerAQLRequest
-	err := apiCom.ReadRequest(r, &queryReqeust)
+	utils.GetRootReporter().GetCounter(utils.QueryReceivedBroker).Inc(1)
+
+	start := utils.Now()
+	var err error
+	defer func() {
+		duration := utils.Now().Sub(start)
+		utils.GetRootReporter().GetTimer(utils.QueryLatencyBroker).Record(duration)
+		if err != nil {
+			utils.GetRootReporter().GetCounter(utils.QueryFailedBroker).Inc(1)
```

Comment:
differentiate some metrics between aql/sql?

Reply:
I don't feel it's necessary to distinguish them.. main purpose for this is to trigger alerts on query failures no matter what caused.
if it's sql parsing error it will be logged.

Reply:
at least add some request count/error count for aql and sql to see how many query go through aql/sql in the future.

Reply:
oh that makes sense. will add separate received counter.